resource-definitions/template-driver/imagepullsecrets/README.md

```diff
@@ -0,0 +1,12 @@
+This section shows how to use the [Template Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/template/) for configuring cluster access to a private container image registry.
+
+The example implements the Kubernetes standard mechanism to [Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). It creates a Kubernetes Secret of `kubernetes.io/dockerconfigjson` type, reading the credentials from a secret store. The example therefore requires the [Humanitec Operator](https://developer.humanitec.com/integration-and-extensions/humanitec-operator/overview/) to be set up for the cluster.
```

Comment:
Should we also mention that it's actually required to do that in Operator Mode as the `Registries` feature doesn't work?

Reply:
Yes, good point. I added a statement accordingly.